recipe/meta.yaml

```diff
@@ -11,22 +11,23 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
+    - setuptools >=45
+    - setuptools-scm >=6.2,<8
     - pip
-    - setuptools_scm
   run:
-    - python >=3.8
+    - python >=3.9
     - numpy
     - scipy
     - bottleneck
     - toolz
     - numexpr
-    - xarray >=2022.9.0
+    - xarray >=2023.1.0
```

Comment:
```suggestion
    - xarray >=2023.9.0
```

Reply:
Yes, I saw that that had been updated after I made the PR.  Let's make that change.